tce/src/tools/Environment.cc

```diff
@@ -1087,8 +1073,27 @@ std::vector<std::string> Environment::implementationTesterTemplatePaths() {
         string path = string(TCE_SRC_ROOT) + DS + "data" + DS + "hdb";
         paths.push_back(path);
     }
-    string path = string(TCE_INSTALLATION_ROOT) + string(INSTALLATION_DIR) 
-        + "data" + DS + "hdb";
+    string path = instPath() + string(INSTALLATION_DIR) + "data" + DS + "hdb";
     paths.push_back(path);
     return paths;
 }
+
+/**
+ * Returns the installation path for TCE
+ * It can be .deb unpacked to homedir (TCE_USER_INSTALL envvar should be set)
+ * or one created by 'make install'
+ *
+ * @return Path to TCE installation root
+ */
+string
+Environment::instPath() {
```

Comment:
There's already a method 'installed()' which doesn't work very robustly. This thus would be better there and initialized in initialize() so this method returns directly contents of a variable. Also rename to installationDir().